src/rpc/mempool.cpp

```diff
@@ -690,12 +697,18 @@ static RPCHelpMan gettxspendingprevout()
                 const CTransaction* spendingTx = mempool.GetConflictTx(prevout);
                 if (spendingTx != nullptr) {
                     o.pushKV("spendingtxid", spendingTx->GetHash().ToString());
+                    if (return_spending_tx) {
```

Comment:
I think you'd want to check `return_spending_tx && *return_spending_tx` here, though maybe just use a boolean directly?

Reply:
Sorry I don't understand this ?

Reply:
I had forgotten this one , fixed in https://github.com/bitcoin/bitcoin/pull/24539/commits/f2a496c827442c7650e5742fdfd49147f0da0143